setup.py

```diff
@@ -27,6 +27,7 @@
             'bump2version = bumpversion.cli:main',
         ]
     },
+    python_requires='>=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*',
```

Comment:
I'm quite new to Python but couldn't that be expressed as `>=2.7, <3.0, >=3.5`?

Reply:
https://www.python.org/dev/peps/pep-0440/#version-specifiers is a bit vague in this regard. The syntax used here in setup.py appears to be quite common from what I've seen. Perhaps this is just people copying each other though :)

Reply:
```python
    python_requires=">=2.7, <3.0, >=3.5",
```
Running `pip install -e .` on this with Python 3.7 gives:

`ERROR: Package 'foo' requires a different Python: 3.7.4 not in '>=2.7, <3.0, >=3.5'`

Reply:
@hugovk Yep. I saw that the `,` is basically a `and`.

Reply:
It's a little bit ugly, all the more reason to drop almost-EOL Python 2.7 and write `python_requires=">=3.5"`!

https://python2woop.pw
https://python3statement.org